<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogImportChecker</class>
 <widget class="QDialog" name="DialogImportChecker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QLabel" name="label_sts">
     <property name="text">
      <string>...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogImportChecker</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogImportChecker</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
